reader-osm/src/test/java/com/graphhopper/GraphHopperIT.java

```diff
@@ -1408,6 +1409,83 @@ public void testDisablingLM() {
         assertTrue(res.getHints().getInt("visited_nodes.sum", 0) > 200);
     }
 
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
+    public void testCompareAlgos(boolean turnCosts) {
```

Comment:
This is nice. I really need to invest some time into JUnit again :)

Reply:
Same here this is about everything I know so far :)